lib/config/presets/internal/monorepo.ts

```diff
@@ -246,6 +246,7 @@ const patternGroups = {
   clarity: ['^@cds/', '^@clr/'],
   embroider: '^@embroider/',
   fullcalendar: '^@fullcalendar/',
+  spfx: ['^@microsoft/sp-', '^@microsoft/eslint-.*-spfx'],
```

Comment:
can it match any false positive package?
```suggestion
  spfx: ['^@microsoft\\/sp-', '^@microsoft\\/eslint-.+-spfx$'],
```